### DISCUSSION TITLE
Do you use Wiki.js full-time at your company? If so, how many employees do you have using it?

### DISCUSSION THREAD

**Discussion by nunick** [requarks/wiki#5013] [General]
I'm interested in implementing a wiki at work (~20 employees) and really like the look of Wiki.js and the dev direction. Has anyone fully implemented it at work as their primary knowledge repo? If so, how large is your company using it?

**Comment by LarsK1** [comment]
Hi,
I use at as standalone information portal. Currently there are 62 fulltime employee working with it ( without any hazzle).

**Reply by nunick** [reply]
@LarsK1 that's great to hear! I appreciate the reply! Is it used to store all company SOPs, projects, and such? Do you use it for any technical procedures as well (manufacturing procedures/maintenance procedures)?

**Reply by LarsK1** [reply]
@nunick Your welcome. I'm a tax adviser - therefore I "only" store notes about servicing the website and customer portals, and the whole internal management (like where to store what). In summary it contains about 1000 pages and the markdown-files only are about 30GB...

**Reply by nunick** [reply]
@LarsK1 that's really interesting! Thank you for sharing. I hope others find it informative too. I'm looking forward to see how others use Wiki.js as well.

**Comment by walthar** [comment]
Hi,
I have it on my organisation for the ICT team of 12 members. We do serve ~1800 end users.
